_posts/2022-03-25-zero-cost-proxies.md

```diff
@@ -0,0 +1,476 @@
+
+*Figure 7:  Relative performance profiles on NATS-Bench TSS and DARTS.*
+
+| TransNASBench-101 | Avg. over all search spaces and tasks|
+| :--: | :--: | 
+| ![Zero-cost PP]({{ site.url }}/public/images/2021-12-01-zero-cost-proxies/pp_trans.png) | ![Zero-cost PP]({{ site.url }}/public/images/2021-12-01-zero-cost-proxies/pp_all_tasks.png) |
+
+*Figure 8: Relative performance profiles on TransNASBench-101, and averaged over all search spaces and tasks.*
+
+### Performance of ZC proxies as training progresses
+
+Finally, we evaluate the stability of ZC proxies as training progresses. We expand on the initial results reported
+by [Dey et al. (2021)](https://arxiv.org/abs/2106.04010) to evaluate each ZC proxies' Spearman 
+rank correlation after every epoch of training on the NATS-Bench TSS search space with two different 
+datasets: CIFAR-10 and Spherical CIFAR-100.  
+
+| Spearmans's rank correlation vs. training epochs <br> NATS-Bench TSS CIFAR-10 | Spearmans's rank correlation vs. training epochs <br> NATS-Bench TSS Spherical CIFAR-100 |
+| :--: | :--: | 
+| ![Zero-cost Epochs]({{ site.url }}/public/images/2021-12-01-zero-cost-proxies/zerocost_epochs_c10.gif) | ![Zero-cost Epochs]({{ site.url }}/public/images/2021-12-01-zero-cost-proxies/zerocost_epochs_scifar100.gif) |
+
+*Figure 9: Spearman's rank correlation vs. training epochs on NATS-Bench TSS CIFAR-10 and Spherical CIFAR-100.*
+
+On CIFAR-10, we find that measures like `snip` and `grad_norm` gradually degrade in rank 
+correlation as the network trains. `jacob_cov` and `grasp` at initialization have rank correlations
+of 0.69 and 0.63 respectively, but after even one epoch of training, they drastically 
+degrade to −0.02 and −0.41. `synflow` stays relatively stable. 
+This is not surprising given that most of the ZC proxies have 
+been directly applied from pruning-at-initialization literature. 
+Similarly, on spherical CIFAR-100, most ZC proxies degrade in performance, except `jacob_cov` and `synflow`.
+
+From both datasets, we observe a counterintuitive property: 
+*ZC proxies are unable to take advantage of strictly increasing
+information*. 
+After all, as a network trains for more epochs, it only gets closer to its final fully-trained state.
+It would not be unreasonable to expect future ZC proxies to be able to capitalize on more information, and so we encourage designers of future ZC proxies to keep this desirable property in mind.
+However, we note that this property can be obtained artificially by combining the ZC proxy with a technique that already has this property.
+For example, there is preliminary work on combining [ZC proxies with learning curve extrapolation](https://arxiv.org/abs/2104.01177).
+
+<!-- Example collapsible text -->
+<!-- <details>
+  <summary>Details of every heatmap</summary>
+
+    Example text.
+
+</details> -->
+
+
```

Comment:
Remove this - its making all of the content below here become the post excerpt in the `blog/` endpoint

Reply:
Done [in this commit](https://github.com/iclr-blog-track/iclr-blog-track.github.io/pull/34/commits/2353c700c914a9f7833e14b5a5e6152336e23ec0)